internal/statemachine/common_states.go

```diff
@@ -133,66 +134,133 @@ func (stateMachine *StateMachine) generateDiskInfo() error {
 	return nil
 }
 
-var calculateRootfsSizeState = stateFunc{"calculate_rootfs_size", (*StateMachine).calculateRootfsSize}
+// Semi-arbitrary value, probably larger than needed but big enough to not have issues
+// and low enough to stay in the same magnitude
+const ext4FudgeFactor = 1.5
```

Comment:
I don't really like putting "ext4" in this name -- as unlikely as it is that we'd make a preinstalled image with a btrfs or xfs or whatever filesystem for /, we'd apply the same multiple I assume?

Presumably this is about giving the rootfs space to grow into as much as anything else?

Reply:
Correct, the idea is to get room for the FS metadata. This is a bit hard to see in this refactoring but I get this name from a comment indicating this is a reasonable value for ext4 FS. There is a bug to try and optimize this (see https://bugs.launchpad.net/ubuntu-image/+bug/1632424). I would like to keep the mention of the FS type in the name to make it very clear this fudge factor is supposed to be adapted to ext4 **only** (even if it could work for others) and the day we support another FS type it should be obvious another value would need to be defined.

Reply:
What you say sounds more like a description of `ext4Padding` than `ext4FudgeFactor`?

Reply:
This is a multiplicative factor, so calling the value `padding` might be confusing, no?